### DISCUSSION TITLE
사용방법 문의

### DISCUSSION THREAD

**Discussion by soooo9** [changdoc/naver-vaccine-macro#131] [General]
안녕하세요.. :)
개발해주신 네이버 백신 프로그램 사용 관련 문의드립니다.
오늘 처음 사용해서 헷갈려가지고요 ㅠㅠ

1) 개별 병원 예약신청 페이지 에서의 자동예약
2) 지도상의 자동 새로고침

두 가지 기능을 동시에 사용해도 되는건가요?
(지도로 자동새로고침 & 한 병원의 예약 신청)

좀전에 동시에 진행을 하던 중 
제가 열어둔 병원 신청페이지가 아닌 새로운 페이지의 예약 페이지에서 예약 시도를 했어요..!
만약 그렇게 될 경우 잘못하면 밴을 먹는 상황이 되는게 아닌가 걱정이 되어서요ㅠㅠ!

**Comment by changdoc** [comment]
지도상에서 자동 새로고침을 하는경우 화면안에 표시되는 병원에서 잔여 백신이 발생했다고 판단되는 경우 해당 병원 예약 신청 페이지가 열리고 예약 신청까지 진행하게 되는데요. 아직 테스트 중인 기능이라 정상 동작 안할 수가 있습니다.

두가지 동시에 사용하게 되면 차단 가능성이 있긴한데 지도 상에서 백신 발생하여 신청 페이지가 열리는 경우가 드물어서 자주 발생하진 않을듯하네요.

**Reply by soooo9** [reply]
오.... 1시간 밴 먹는게 치명적이긴 한데 달콤하긴 하네요ㅠㅠㅠ
6시 넘은 한적한 시간대라 가능했던거같아요...!

그럼 지도 자동 새로고침 하지 않고 병원의 자동예약만 이용해도 문제는 없는걸까요?
아니면 동시에 진행하는게 맞는걸까요..?!
